testing/sdk_utils.py

```diff
@@ -1,4 +1,5 @@
 import sys
+
```

Comment:
Ha! Yeah I noticed that, I'll pull it. I had put a ton of code in there, pulled it out to `sdk_repository.py` and that change was left after the imports got cleaned out.